Resources/Textures/Objects/Misc/monkeycube.rsi/meta.json

```diff
@@ -1,7 +1,7 @@
 {
   "version": 1,
   "license": "CC-BY-SA-3.0",
-  "copyright": "Taken from cev-eris at https://github.com/discordia-space/CEV-Eris/raw/9c980cb9bc84d07b1c210c5447798af525185f80/icons/obj/food.dmi. box_kobold, box_variant and wrapper_kobold are edited versions of the raw sprites by Ubaser.",
+  "copyright": "Taken from cev-eris at https://github.com/discordia-space/CEV-Eris/raw/9c980cb9bc84d07b1c210c5447798af525185f80/icons/obj/food.dmi. box_kobold, box_variant and wrapper_kobold are edited versions of the raw sprites by Ubaser. box_moproach is hue changed version of box",
```

Comment:
I guess technically you gotta include that you're the person who did the hue change for it to be proper attribution, I think.